sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -57,14 +59,25 @@
  *
  * pipeline.apply(JdbcIO.read()
  *   .withDataSource(myDataSource)
+ *   .withRowMapper(new JdbcIO.RowMapper<MyElement>() {
+ *     public MyElement mapRow(ResultSet resultSet) {
+ *       // use the resultSet to build the element of the PCollection
+ *       // for instance:
+ *       // return resultSet.getString(2);
```

Comment:
This line should not be commented out; and it should return a MyElement.
E.g.: return new MyElement(resultSet.getInt(1), resultSet.getString(2)) or something
(it should also correspond to the query - internal inconsistency within an example can be confusing to readers of the documentation)